src/Sql/Sql/help/Disable-AzSqlDatabaseLedgerDigestUpload.md

```diff
@@ -0,0 +1,185 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
```

Comment:
Do we also need to update the documentation of the cmdlet to create a SQL Database to explain our new parameter?

Reply:
yes, I think I need to manually do this one